modules/custom/openy_hours_formatter/js/openy_hours_formatter.js

```diff
@@ -22,4 +23,89 @@
       });
     }
   };
+
+  Drupal.behaviors.today_hours = {
+
+    /**
+     * Used to track the setInveral.
+     */
+    refreshTimer: false,
+
+    /**
+     * See https://moment.github.io/luxon/docs/manual/formatting.
+     * @returns {string}
+     */
+    getDayOfWeek(tz) {
+      return luxon.DateTime.local().setZone(tz).toFormat('c');
+    },
+
+    /**
+     * See https://moment.github.io/luxon/docs/manual/formatting.
+     * @returns {string}
+     */
+    getDate(tz) {
+      return luxon.DateTime.local().setZone(tz).toFormat('yyyy-LL-dd');
+    },
+
+    /**
+     * Primary method for updating the today hours.
+     */
+    updateTodayHours() {
+      var $todayHours = $('.today-hours > .today');
+      var hoursData = [];
+      var branchHours = drupalSettings.openy_hours_formatter.branch_hours || [];
```

Comment:
what if ```openy_hours_formatter``` is not exist on a page?
Let's have a check to eliminate JS errors.

Reply:
Will do.